src/lib/auth/github/permissions.js

```diff
@@ -5,7 +5,8 @@ const axios = require('axios')
 const config = require('../../../config')
 const logger = require('../../logger')
 
-const GITHUB_TEAMS_API_ENDPOINT = 'https://api.github.com/teams'
+const ALPHAGOV_GITHUB_ORGANISATION_ID = 596977
+const GITHUB_TEAMS_API_ENDPOINT = `https://api.github.com/organizations/${ALPHAGOV_GITHUB_ORGANISATION_ID}/team`
 
 const validateUserTeamMembership = async function validateUserTeamMembership(user, token, team) {
   const url = `${GITHUB_TEAMS_API_ENDPOINT}/${team}/members/${user}`
```

Comment:
That would give us `https://api.github.com/organizations/596977/team/orgs/596977/${team}/memberships/${user}`, which is probably not what you intended!

There’s a difference between `/organizations/:org_id` for ID-based access and `/orgs/:org` for name-based access. Since we’re already using IDs for the teams, we probably want to stick to ID-based access.

But I’ll try to tweak it so the URL is less broken up.